### ISSUE TITLE
[FEATURE] auto web auto trap

### ISSUE THREAD

**Issue by Magiaaa** [CCBlueX/LiquidBounce#3733]
I want you to add auto web what web player and auto trap what place blocks around player what we can add custom blocks like dirt planks obsidian

**Comment by superblaubeere27** [comment]
I think we should merge this with Ignite.

**Comment by 1zun4** [comment]
Yes, that would make sense. I will make one AutoTrap module with the different selections of what you want to do.

**Comment by Magiaaa** [comment]
what is ignite

czw., 22 sie 2024 o 15:48 1zuna ***@***.***> napisał(a):

**Comment by Evidentiallll** [comment]
It should also make sure it places on blocks (unlike meteor's which loves to place mid-air) and I would also like a delay option because the server I play on (Hoplite) does not like instant-place
